### ISSUE TITLE
django-channels support

### ISSUE THREAD

**Issue by amureki** [jazzband/django-configurations#155]
As you know, django-channels is on its way to Django 1.10, so I wonder, is that possible to combine configurations app with channels [asgi.py](http://channels.readthedocs.io/en/latest/deploying.html#run-interface-servers)?
Can we update documentation for this case?

**Comment by fpischedda** [comment]
Hi, 

I don't know if you already solved this but I've found a quick workarond for the issue

In your asgi.py module add:

```
from configurations import importer

# set your env defaults

importer.install()

channel_layer = get_channel_layer()
```

**Comment by amureki** [comment]
@fpischedda nope, I just removed django-configurations from my project, it is anyways very small and simple: https://github.com/amureki/lunch-with-channels

But thank you for finding this out!

**Comment by blueyed** [comment]
I think it should be added to the doc.
Reopening.

@fpischedda 
Would you like to create a PR?

**Comment by fpischedda** [comment]
@blueyed 
yes, thank you :)
